### PR TITLE
Minor tweaks and a disctl example.

### PR DESCRIPTION
**Pull request by activeshadow**

### CODE REVIEW

File: src/cmds/ldpcap/host.go [hunk 1]
```diff
@@ -133,8 +133,6 @@ func (h *Host) Write(out io.Writer) {
 	if v := fmtStrings(h.Routers); v != "" {
 		fmt.Fprintf(out, "routers=%v\n", v)
 	}
-
-	fmt.Fprintln(out, "\n")
```
Comment (activeshadow) [inline src/cmds/ldpcap/host.go:137]: This was causing a failure when running tests, saying something along the lines of a redundant newline was being created. If a newline is really needed here, I can replace `Fprintln` with `Fprint`.

File: examples/disctl/build.bash [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+# Create the first endpoint.
+endpoint=$(bin/disctl -ne)
+
+# Add the first endpoint to the network.
+bin/disctl -c $net1 $endpoint
+
+# Create the second endpoint.
+endpoint=$(bin/disctl -ne)
+
+# Add the second endpoint to the network.
```
Comment (jcrussell) [inline examples/disctl/build.bash]: Do you want to change this to be a for loop? Something like:

```bash
for i in $(seq 2); do
	# Create an endpoint.
	endpoint=$(bin/disctl -ne)
	# Add it to the network.
	bin/disctl -c $net1 $endpoint
done
```

This makes it clearer how you might extend this to many nodes.

**Review by jcrussell — COMMENTED** [review]
Looks pretty good, thanks! Don't forget to add yourself to CONTRIBUTORS.

File: examples/disctl/build.bash (hunk 2 above)
Comment (activeshadow) [reply examples/disctl/build.bash]: Great idea. Can do!

File: src/cmds/ldpcap/host.go (hunk 1 above)
Comment (activeshadow) [reply src/cmds/ldpcap/host.go:137]: @jcrussell thoughts on this? Can I leave it out or should I replace it with `Fprint` (or `Fprintln` without the `\n')?

Comment (jcrussell) [reply src/cmds/ldpcap/host.go:137]: Yeah, we should probably add an extra newline somehow. Was it a `go vet` complaint?

**Comment by activeshadow** [comment]
@jcrussell take one more look if you want to. I'd like to squash before you merge though, unless that's not the normal MO.

**Comment by jcrussell** [comment]
Looks good! You can squash the history if you'd like, we don't typically though.

**Comment by activeshadow** [comment]
@jcrussell I'll leave the history since you don't typically squash. Merge away!

**Pull request merged**